src/celengine/stardb.cpp

```diff
@@ -754,7 +732,7 @@ void StarDatabase::finish()
 static void stcError(const Tokenizer& tok,
                      const string& msg)
 {
-    fmt::fprintf(cerr,  _("Error in .stc file (line %i): %s\n"), tok.getLineNumber(), msg);
+    fmt::fprintf(cout,  _("Error in .stc file (line %i): %s\n"), tok.getLineNumber(), msg);
```

Comment:
in final version please revert this change

Reply:
I was sure I already reverted it... :open_mouth: